utils/generate_types/overrides.d.ts

```diff
@@ -168,7 +167,7 @@ export interface CDPSession {
   ): Promise<Protocol.CommandReturnValues[T]>;
 }
 
-type DeviceDescriptor = {viewport: BrowserNewContextOptionsViewport, userAgent: string};
+type DeviceDescriptor = {viewport: ViewportSize, userAgent: string};
```

Comment:
I think device descriptors also have isMobile, deviceScaleFactor and hasTouch.

Reply:
True, this has been wrong. I'll fix in a separate patch. I don't want to change the exported API in this one.